<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="10" total="20" passed="10">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="2743776" started-at="2018-08-02T10:21:02Z" finished-at="2018-08-02T11:06:46Z">
    <groups>
    </groups>
    <test name="Internet Explorer Browser" duration-ms="2743776" started-at="2018-08-02T10:21:02Z" finished-at="2018-08-02T11:06:46Z">
      <class name="com.tcs.BsiShopRedesign.sprint8.VerifyHardCopyDetails">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyHardCopyDetails@7276c8cd]" name="launchBrowser" is-config="true" duration-ms="8012" started-at="2018-08-02T11:00:45Z" finished-at="2018-08-02T11:00:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="signIn()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyHardCopyDetails@7276c8cd]" name="signIn" duration-ms="23869" started-at="2018-08-02T11:00:53Z" finished-at="2018-08-02T11:01:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method status="FAIL" signature="verifyHardCopyDeliveryDetails()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyHardCopyDetails@7276c8cd]" name="verifyHardCopyDeliveryDetails" duration-ms="328929" started-at="2018-08-02T11:01:17Z" finished-at="2018-08-02T11:06:46Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Placing of order was unsuccesful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Placing of order was unsuccesful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.CheckoutPage.confirmOrderDetails(CheckoutPage.java:163)
	at com.tcs.BsiShopRedesign.sprint8.VerifyHardCopyDetails.verifyHardCopyDeliveryDetails(VerifyHardCopyDetails.java:97)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHardCopyDeliveryDetails -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyHardCopyDetails@7276c8cd]" name="tearDown" is-config="true" duration-ms="579" started-at="2018-08-02T11:06:46Z" finished-at="2018-08-02T11:06:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint8.VerifyHardCopyDetails -->
      <class name="com.tcs.BsiShopRedesign.sprint7.CreateCheckoutBillingAddress">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.CreateCheckoutBillingAddress@782663d3]" name="launchBrowser" is-config="true" duration-ms="16444" started-at="2018-08-02T10:31:11Z" finished-at="2018-08-02T10:31:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="signIn()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.CreateCheckoutBillingAddress@782663d3]" name="signIn" duration-ms="23304" started-at="2018-08-02T10:31:27Z" finished-at="2018-08-02T10:31:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method status="FAIL" signature="editBillingAddress()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.CreateCheckoutBillingAddress@782663d3]" name="editBillingAddress" duration-ms="315437" started-at="2018-08-02T10:31:50Z" finished-at="2018-08-02T10:37:06Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Placing of order was unsuccesful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Placing of order was unsuccesful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.CheckoutPage.confirmOrderDetails(CheckoutPage.java:163)
	at com.tcs.BsiShopRedesign.sprint7.CreateCheckoutBillingAddress.editBillingAddress(CreateCheckoutBillingAddress.java:89)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- editBillingAddress -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.CreateCheckoutBillingAddress@782663d3]" name="tearDown" is-config="true" duration-ms="597" started-at="2018-08-02T10:37:06Z" finished-at="2018-08-02T10:37:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint7.CreateCheckoutBillingAddress -->
      <class name="com.tcs.BsiShopRedesign.sprint8.VerifyMemberDetails">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyMemberDetails@64485a47]" name="launchBrowser" is-config="true" duration-ms="7721" started-at="2018-08-02T10:42:17Z" finished-at="2018-08-02T10:42:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="signInMember()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyMemberDetails@64485a47]" name="signInMember" duration-ms="14016" started-at="2018-08-02T10:42:25Z" finished-at="2018-08-02T10:42:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInMember -->
        <test-method status="PASS" signature="verifyMemberLink()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyMemberDetails@64485a47]" name="verifyMemberLink" duration-ms="61750" started-at="2018-08-02T10:42:39Z" finished-at="2018-08-02T10:43:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyMemberLink -->
        <test-method status="PASS" signature="enterEmailID()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyMemberDetails@64485a47]" name="enterEmailID" duration-ms="18272" started-at="2018-08-02T10:43:41Z" finished-at="2018-08-02T10:43:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- enterEmailID -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.VerifyMemberDetails@64485a47]" name="tearDown" is-config="true" duration-ms="596" started-at="2018-08-02T10:43:59Z" finished-at="2018-08-02T10:44:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint8.VerifyMemberDetails -->
      <class name="com.tcs.BsiShopRedesign.sprint7.VerifyMemberDiscount">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.VerifyMemberDiscount@4fb64261]" name="launchBrowser" is-config="true" duration-ms="13392" started-at="2018-08-02T10:26:26Z" finished-at="2018-08-02T10:26:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="signInMember()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.VerifyMemberDiscount@4fb64261]" name="signInMember" duration-ms="15754" started-at="2018-08-02T10:26:39Z" finished-at="2018-08-02T10:26:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInMember -->
        <test-method status="FAIL" signature="verifyMemberPrice()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.VerifyMemberDiscount@4fb64261]" name="verifyMemberPrice" duration-ms="47338" started-at="2018-08-02T10:26:55Z" finished-at="2018-08-02T10:27:42Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The member price in the Basket Page is incorrect]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The member price in the Basket Page is incorrect
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.BasketPage.verifyMemPriceBasket(BasketPage.java:465)
	at com.tcs.BsiShopRedesign.sprint7.VerifyMemberDiscount.verifyMemberPrice(VerifyMemberDiscount.java:82)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyMemberPrice -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.VerifyMemberDiscount@4fb64261]" name="tearDown" is-config="true" duration-ms="609" started-at="2018-08-02T10:27:42Z" finished-at="2018-08-02T10:27:43Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint7.VerifyMemberDiscount -->
      <class name="com.tcs.BsiShopRedesign.sprint8.RenewMembership">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.RenewMembership@25bbf683]" name="launchBrowser" is-config="true" duration-ms="7650" started-at="2018-08-02T10:44:00Z" finished-at="2018-08-02T10:44:07Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="signInMember()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.RenewMembership@25bbf683]" name="signInMember" duration-ms="13268" started-at="2018-08-02T10:44:07Z" finished-at="2018-08-02T10:44:21Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signInMember -->
        <test-method status="FAIL" signature="addRenewMembershipToBasket()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.RenewMembership@25bbf683]" name="addRenewMembershipToBasket" duration-ms="297402" started-at="2018-08-02T10:44:21Z" finished-at="2018-08-02T10:49:18Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Placing of order was unsuccesful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Placing of order was unsuccesful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.CheckoutPage.confirmOrderDetails(CheckoutPage.java:163)
	at com.tcs.BsiShopRedesign.sprint8.RenewMembership.addRenewMembershipToBasket(RenewMembership.java:76)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addRenewMembershipToBasket -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.RenewMembership@25bbf683]" name="tearDown" is-config="true" duration-ms="586" started-at="2018-08-02T10:49:18Z" finished-at="2018-08-02T10:49:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint8.RenewMembership -->
      <class name="com.tcs.BsiShopRedesign.sprint7.EditCheckoutBillingAddress">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.EditCheckoutBillingAddress@42607a4f]" name="launchBrowser" is-config="true" duration-ms="7703" started-at="2018-08-02T10:27:43Z" finished-at="2018-08-02T10:27:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="FAIL" signature="signIn()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.EditCheckoutBillingAddress@42607a4f]" name="signIn" duration-ms="43157" started-at="2018-08-02T10:27:50Z" finished-at="2018-08-02T10:28:34Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Login UnSuccessful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Login UnSuccessful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.SignInPage.login(SignInPage.java:71)
	at com.tcs.BsiShopRedesign.sprint7.EditCheckoutBillingAddress.signIn(EditCheckoutBillingAddress.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method status="FAIL" signature="editBillingAddress()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.EditCheckoutBillingAddress@42607a4f]" name="editBillingAddress" duration-ms="156444" started-at="2018-08-02T10:28:34Z" finished-at="2018-08-02T10:31:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Edit Address in Checkout Page was unsuccessful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Edit Address in Checkout Page was unsuccessful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.sprint7.EditCheckoutBillingAddress.editBillingAddress(EditCheckoutBillingAddress.java:102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- editBillingAddress -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.EditCheckoutBillingAddress@42607a4f]" name="tearDown" is-config="true" duration-ms="622" started-at="2018-08-02T10:31:10Z" finished-at="2018-08-02T10:31:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint7.EditCheckoutBillingAddress -->
      <class name="com.tcs.BsiShopRedesign.sprint8.ApplyMemberAccount">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.ApplyMemberAccount@1990a65e]" name="launchBrowser" is-config="true" duration-ms="7761" started-at="2018-08-02T10:37:06Z" finished-at="2018-08-02T10:37:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="signIn()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.ApplyMemberAccount@1990a65e]" name="signIn" duration-ms="24673" started-at="2018-08-02T10:37:14Z" finished-at="2018-08-02T10:37:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method status="FAIL" signature="applyMemberAccountBelowBandNine()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.ApplyMemberAccount@1990a65e]" name="applyMemberAccountBelowBandNine" duration-ms="222900" started-at="2018-08-02T10:37:39Z" finished-at="2018-08-02T10:41:22Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Remove product from Basket was unsuccessful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Remove product from Basket was unsuccessful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.BasketPage.removeProduct(BasketPage.java:216)
	at com.tcs.BsiShopRedesign.sprint8.ApplyMemberAccount.applyMemberAccountBelowBandNine(ApplyMemberAccount.java:91)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- applyMemberAccountBelowBandNine -->
        <test-method status="PASS" signature="applyMemberAccountBandNineAbove()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.ApplyMemberAccount@1990a65e]" name="applyMemberAccountBandNineAbove" duration-ms="54880" started-at="2018-08-02T10:41:22Z" finished-at="2018-08-02T10:42:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- applyMemberAccountBandNineAbove -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.ApplyMemberAccount@1990a65e]" name="tearDown" is-config="true" duration-ms="594" started-at="2018-08-02T10:42:17Z" finished-at="2018-08-02T10:42:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint8.ApplyMemberAccount -->
      <class name="com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads">
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads@6ec8211c]" name="launchBrowser" is-config="true" duration-ms="17409" started-at="2018-08-02T10:49:19Z" finished-at="2018-08-02T10:49:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="PASS" signature="signIn()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads@6ec8211c]" name="signIn" duration-ms="23294" started-at="2018-08-02T10:49:36Z" finished-at="2018-08-02T10:49:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method status="FAIL" signature="downloadSoftCopyfromOrderCofirmationPage()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads@6ec8211c]" name="downloadSoftCopyfromOrderCofirmationPage" duration-ms="336417" started-at="2018-08-02T10:49:59Z" finished-at="2018-08-02T10:55:36Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Placing of order was unsuccesful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Placing of order was unsuccesful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.CheckoutPage.confirmOrderDetails(CheckoutPage.java:163)
	at com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads.downloadSoftCopyfromOrderCofirmationPage(SoftCopyDownloads.java:104)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- downloadSoftCopyfromOrderCofirmationPage -->
        <test-method status="FAIL" signature="downloadSoftCopyfromOrderHistoryPage()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads@6ec8211c]" name="downloadSoftCopyfromOrderHistoryPage" duration-ms="308550" started-at="2018-08-02T10:55:36Z" finished-at="2018-08-02T11:00:44Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[SoftCopy Download was unsuccessful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: SoftCopy Download was unsuccessful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.OrderHistoryPage.downloadAndVerifySoftCopy(OrderHistoryPage.java:106)
	at com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads.downloadSoftCopyfromOrderHistoryPage(SoftCopyDownloads.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- downloadSoftCopyfromOrderHistoryPage -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads@6ec8211c]" name="tearDown" is-config="true" duration-ms="594" started-at="2018-08-02T11:00:44Z" finished-at="2018-08-02T11:00:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint8.SoftCopyDownloads -->
      <class name="com.tcs.BsiShopRedesign.sprint7.UnregisteredUserCheckout">
        <test-method status="PASS" signature="getDriverInstance()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.UnregisteredUserCheckout@43738a82]" name="getDriverInstance" is-config="true" duration-ms="16" started-at="2018-08-02T10:21:02Z" finished-at="2018-08-02T10:21:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDriverInstance -->
        <test-method status="PASS" signature="launchBrowser(java.lang.String)[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.UnregisteredUserCheckout@43738a82]" name="launchBrowser" is-config="true" duration-ms="17170" started-at="2018-08-02T10:21:02Z" finished-at="2018-08-02T10:21:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[IE]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="FAIL" signature="CheckLoginAlertOnChekout()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.UnregisteredUserCheckout@43738a82]" name="CheckLoginAlertOnChekout" duration-ms="305223" started-at="2018-08-02T10:21:20Z" finished-at="2018-08-02T10:26:25Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Placing of order was unsuccesful]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Placing of order was unsuccesful
	at org.testng.Assert.fail(Assert.java:94)
	at com.tcs.BsiShopRedesign.utilities.CommonHelper.reportFailure(CommonHelper.java:67)
	at com.tcs.BsiShopRedesign.pages.CheckoutPage.confirmOrderDetails(CheckoutPage.java:163)
	at com.tcs.BsiShopRedesign.sprint7.UnregisteredUserCheckout.CheckLoginAlertOnChekout(UnregisteredUserCheckout.java:67)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CheckLoginAlertOnChekout -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.tcs.BsiShopRedesign.sprint7.UnregisteredUserCheckout@43738a82]" name="tearDown" is-config="true" duration-ms="854" started-at="2018-08-02T10:26:25Z" finished-at="2018-08-02T10:26:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.tcs.BsiShopRedesign.sprint7.UnregisteredUserCheckout -->
    </test> <!-- Internet Explorer Browser -->
  </suite> <!-- Suite -->
</testng-results>
